builder/hyperv/common/step_set_boot_order.go

```diff
@@ -0,0 +1,37 @@
+package common
```

Comment:
I recommend writing a test for this using the `builder/hyperv/common/driver_mock.go`

Reply:
It now has a simple test.